ui-components/src/blocks/InfoCard.docs.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs"
+import { Swatch } from "../prototypes/Swatch"
+import { InfoCard } from "./InfoCard"
+
+# Info Card
+
+The Info Card component renders a box with a title that can be text of a link. Beneath the title is an optional subtitle. Beneath the subtitle can be a string, a component, or markdown.
```

Comment:
`can be text of a link` might be a typo here?

Reply:
fixed!